init.fish

```diff
@@ -41,16 +41,20 @@ source $OMF_CONFIG/before.init.fish ^/dev/null
 set -l user_function_path $fish_function_path[1]
 set fish_function_path[1] $OMF_PATH/lib
 
-set -l theme {$OMF_PATH,$OMF_CONFIG}/themes/(cat $OMF_CONFIG/theme)
+# Autoload util functions
+autoload $OMF_PATH/lib $OMF_PATH/lib/git
 
-for path in $OMF_PATH/lib $OMF_PATH/lib/git {$OMF_PATH,$OMF_CONFIG}/pkg/* $theme
-  contains -- (basename $path) $OMF_IGNORE; and continue
+for path in {$OMF_PATH,$OMF_CONFIG}/pkg/*
+  set -l name (basename $path)
 
-  autoload $path $path/completions
-  source $path/(basename $path).fish ^/dev/null
-    and emit init_(basename $path) $path
+  contains -- $name $OMF_IGNORE; and continue
+  require $name
 end
 
+# Autoload theme
+autoload {$OMF_PATH,$OMF_CONFIG}/themes/(cat $OMF_CONFIG/theme)
```

Comment:
```
function autoload_one -d "autoload the first directory which exists in the list of directories"
  for path in $argv
    autoload $path; and return
  end
end
```

then we could

```
autoload_one {$OMF_CONFIG,$OMF_PATH}/themes/(cat $OMF_CONFIG/theme)
```

?

(note that the OMF_PATH/OMF_CONFIG are swapped, so the local one wins if two have the same name)